specs/validator/tableland-openapi-spec.yaml

```diff
@@ -198,43 +227,38 @@ components:
             }
         schema:
           $ref: "#/components/schemas/Schema"
-    TxnReceipt:
+    TransactionReceipt:
       type: object
       properties:
-        tableId:
+        table_id:
           type: string
           example: healthbot_5_1
-        txnHash:
+        transaction_hash:
           type: string
           example: "0x400508d7cc035b14cc53f64393a8dafcc55f66ad8f9b44d626744157337e2098"
-        blockNumber:
+        block_number:
           type: integer
           format: int64
           example: 1
-        chainId:
+        chain_id:
           type: integer
           format: int32
           example: 80001
-    Info:
```

Comment:
This `Info` object is a left over of removing the "get by structure hash" API that was removed.